r2/r2/models/link.py

```diff
@@ -408,9 +408,6 @@ def slug():
             res = "/%s" % p
         return res
 
-    def make_permalink_slow(self):
-        return self.make_permalink(self.subreddit_slow)
-
```

Comment:
It looks like you deleted `Link#comment`, assuming that `Comment#comment` was a duplicate method. There are many class definitions in that file.